src/assembler/parser/index.ts

```diff
@@ -5,8 +5,9 @@ import { ITextMatch, ITextParseRule, parseText } from './text'
 const SYMBOL = `[a-z_]+[a-z0-9_]*|\\|[a-z0-9._ ]+\\|`
 const VALUE = `[0-9a-z#]+`
 const SPACE_OR_TAB = `[ \\t]`
+const STRING = `("(?:[^'"\n]|'"?)*")`
 
-const OPTION = `(?:[0-9a-z#=_]|[\\[{]${SPACE_OR_TAB}*|${SPACE_OR_TAB}*[\\]}]|${SPACE_OR_TAB}*-${SPACE_OR_TAB}*)+`
+const OPTION = `((?:[0-9a-z#=_]|[\\[{]${SPACE_OR_TAB}*|${SPACE_OR_TAB}*[\\]}]|${SPACE_OR_TAB}*-${SPACE_OR_TAB}*)+|${STRING})`
```

Comment:
Please make regex groups non-matching if not used within on match function

Reply:
Done